openedx/core/djangoapps/programs/management/commands/backpopulate_program_credentials.py

```diff
@@ -38,13 +40,53 @@ def add_arguments(self, parser):
             default=False,
             help='Submit tasks for processing.'
         )
+        parser.add_argument(
+            '--args-from-database',
+            action='store_true',
+            default=False,
+            help='Use arguments from the Config model instead of the command line.',
+        )
+        parser.add_argument(
+            '--programs',
```

Comment:
Also change this to `--program-uuids`

Reply:
changed the `programs` parameter to `program-uuids`.